website/content/en/docs/setup/going-to-prod/sizing.md

```diff
@@ -0,0 +1,154 @@
+
+{{< info >}}
+More information about vertical sizing can be found in the [capacity planning section](#capacity-planning).
+{{< /info >}}
+
+### Horizontal Scaling
+
+Fronting your Vector instances with a load balancer achieves horizontal scaling.
+
+![Load balancing example](/img/going-to-prod/horizontal-scaling.png)
+
+#### Choosing a Load Balancer
+
+Choose the load balancer you’re most comfortable operating in a highly available manner. When in doubt, use a managed load balancer, such as AWS ALB. These load balancers are highly available and easier to integrate within your cloud environment.
+
+#### Configuring a Load Balancer
+
+When configuring a load balancer, we recommend the following general settings:
+
+- [Choose a protocol](/docs/setup/going-to-prod/architecting/#protocols) between your clients that allows for even load-balancing and application-level acknowledgement. Such as an HTTP-based protocol.
+- Enable keep-alive for both clients and servers.
+- If you’re using stateful transforms, such as the `aggregate` or `dedupe` transforms, use a load balancing algorithm that consistently routes clients to the same server (i.e., the HAProxy `source` algorithm).
+- If you’re not using stateful transforms, use a load balancing algorithm that evenly distributes traffic (i.e., the HAProxy `roundrobin` algorithm).
+- Configure load balancers to use Vector’s `/health` API endpoint for automatically excluding unresponsive Vector servers.
+- Ensure that load balancers automatically register all targets as your aggregators scale up and down.
+
+#### Avoiding Hot Spots
+
+Not all connections are equal; some connections produce much more data making it difficult to evenly load balance traffic across your aggregators. To mitigate this, we recommend the following best practices:
+
+- Use a protocol that allows for even load balancing, such as an HTTP-based protocol. Avoid plain TCP or UDP connections for the same reasons.
+- Distribute data across multiple connections for easier load balancing.
+- Ensure your Vector instances are large enough to handle your highest volume connection to take full advantage of vertical scaling.
+- Avoid stateful transformation in aggregators (i.e., the `aggregate` transform), when possible, so that you can use a more fair balancing algorithm.
+
+### Autoscaling
+
+For the vast majority of Vector deployments, autoscaling should be based on average CPU utilization. Vector is almost always CPU constrained and CPU utilization is the strongest signal for autoscaling since it will not produce false positives. We recommend the following settings, adjust as necessary:
+
+- Average CPU over 5 minutes with a 85% utilization target.
+- A 5 minute stabilization period for scaling up and down.
+
+These settings are the default in platforms that support them, such as Kubernetes via Helm.
```

Comment:
Noting that we'll need to make that change to have it be the default for our chart

Reply:
Agreed, I'd drop this note until we update the chart, unless Spencer feels like updating the chart now 😄 .